contextual-inquiries/contextual_inquiry_1.typ

```diff
@@ -0,0 +1,198 @@
+Sie achtet sich auf darauf, welche Module in der Liste zur Modulanmeldung rot oder gelb hinterlegt sind.
+In der 2. Phase fragt sie allenfalls auch bei Dozenten direkt nach, ob diese beabsichtigen die Vorlesung zum Modul zu streamen oder andersweitig aufzuzeichnen.
+
+/ Wo holst du dir diese Infos?:
+Infos zu Module über Adunis.
+Meinung und Erfahrungen zu Module über Kollegen, die es bereits besucht haben.
+Sie wusste, dass man über ein Modul im Semesterplaner zu dessen Beschreibung auf Adunis gelangen kann.
+
+/ Holst du Infos zu den Modulabhängigkeiten? Wenn ja, wo?:
+Schaut die "Empfohlene Module" auf Adunis an.
+
+/ Holst du dir Infos zu den Semestern/Moduldurchführungen? Wo?:
+Schaut die Durchführung auf Adunis nach.
+
+/ Wie planst du die Erreichung einer Vertiefung?:
+Gar nicht mehr.
+
+/ Wie planst du die Erfüllung der notwendigen Credits pro Kategorie?:
+[Planer]
+
+/ Hast du ein Zielmodul am Horizont?:
+Sie möchte gerne Compilerbau belegen. Dazu schaut sie sich die "Empfohlene Module" an.
+
+/ Hältst du dich an zuvor empfohlene Module? Wie planst du diese ein?:
+Sie plant Semester grob nach dem Musterstudienplan.
+
+/ Was ist mühsam bei der Recherche und weshalb?:
+Adunis ist zu mühsam, um Module zu finden.
+Die Liste der Module bei der Modulanmeldung enthält zu viele Module und kann nicht gefiltert werden.
+Sie würde gerne Module nach dere Kategorie, Thema oder "Empfohlene Module" filtern.
+
+/ Was ist einfach bei der Recherche und weshalb?:
+Sie konsultiert gerne mit einer Kollegin, die bei der Planung hilfreiche Tipps gibt.
+
+==== Semesterplaner anpassen
+
+/ Wie findest du Module im Planer?:
+Sucht Module nach Namen.
+Suche nach Kürzel funktioniert bei ihr auf Chrome ebenfalls.
+Nutzt DragNDrop selten.
+
+/ Auf welchen Geräten nutzt du den Planer/passt du den Planer an?:
+Laptop und Desktop.
+
+/ Wie validierst du die Studierbarkeit deiner Modulauswahl?:
+Ihr war nicht bewusst, dass Module in Semestern hinzugefügt werden können, in denen sie gar nicht angeboten werden.
+Da sie Module anhand vom Musterstudienplan und der Liste bei der Modulanmeldung wählt, war dies jedoch nie ein Problem.
+
+/ Wie erweiterst du die Semesterauswahl?:
+[Nichts]
+
+/ Welche sichtbaren Informationen sind relevant und weshalb?:
+Die Erfüllung der Kategorien ist ihr wichtig.
+Sie interessieren nur das jetzige und die nächsten Semester, die vorherigen nicht.
+Wenn sie jedoch die besuchten Semester nachschauen will, bevorzugt sie den Semesterplaner, da sie dort nach Semester gruppiert und farblich markiert sind.
+Sie mag, dass die Reihenfolge der Module in der URL persistiert werden.
+
+/ Wer denkst du, steht hinter lost.university?:
+Es wurde bereits bei der Empfehlung erwähnt, dass es ein Studenten-Projekt ist.
+Sie weiss dank dem Footer auch, wer diese Studenten sind.
+
+==== Plan speichern
+
+/ Wie persistierst du deinen gemachten Plan?:
+Sie speichert sich die URL ab.
+[One Drive?]
```

Comment:
yes, er speichert den link in seinem OneDrive (weiss nicht ob persönlich oder von der OST, könnten wir noch nachfragen)

Reply:
Aber glichziitig hani au notiert, dass er es Bookmark het. Do bini liecht verwirrt. Ich find ich im Video grad d Stell nüm, wo er erklärt, wie er ihn speichert.
Ich hau ihn eifach rasch ah und froge.